### PR TITLE
fix: add integrate testing for puptoo

### PR DESCRIPTION
**Pull request by chenlizhong**
add integrate tests for puptoo app

### CODE REVIEW

File: auto-generated/cluster/stone-prd-rh01/tenants/insights-core-tenant/appstudio.redhat.com_v1beta1_integrationtestscenario_insights-puptoo-tekton-testing.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apiVersion: appstudio.redhat.com/v1beta1
```
Comment (jsztuka) [inline auto-generated/cluster/stone-prd-rh01/tenants/insights-core-tenant/appstudio.redhat.com_v1beta1_integrationtestscenario_insights-puptoo-tekton-testing.yaml:1]: I recommend using latest version of crd which is **v1beta2**.

File: cluster/stone-prd-rh01/tenants/insights-core-tenant/insights-puptoo-app-bonfire-tekton.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+apiVersion: appstudio.redhat.com/v1beta1
```
Comment (jsztuka) [inline cluster/stone-prd-rh01/tenants/insights-core-tenant/insights-puptoo-app-bonfire-tekton.yaml:1]: I recommend using latest version of crd which is v1beta2.

Comment (chenlizhong) [reply cluster/stone-prd-rh01/tenants/insights-core-tenant/insights-puptoo-app-bonfire-tekton.yaml]: Updated

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]
something is wrong. Why were the secrets deleted?

**Comment by chenlizhong** [comment]
Error: Failed to read kustomization file under /Users/chenlizhong/work2024/tenants-config/cluster/stone-prd-rh01/tenants/insights-core-tenant:
apiVersion for Kustomization should be kustomize.config.k8s.io/v1beta1

**Comment by chenlizhong** [comment]
@jsztuka  Have to use v1beta1 as error during build-manifests.sh

**Comment by chenlizhong** [comment]
@jsztuka  @ernesgonzalez33   Please review

**Review by jsztuka — APPROVED** [review]
lgtm

**Review by ernesgonzalez33 — APPROVED** [review]

File: auto-generated/cluster/stone-prd-rh01/tenants/insights-core-tenant/appstudio.redhat.com_v1beta2_integrationtestscenario_insights-puptoo-tekton-testing.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  labels:
+    test.appstudio.openshift.io/optional: "true"
+  name: insights-puptoo-tekton-testing
+  namespace: insights-rules-tenant
```
Comment (ernesgonzalez33) [inline auto-generated/cluster/stone-prd-rh01/tenants/insights-core-tenant/appstudio.redhat.com_v1beta2_integrationtestscenario_insights-puptoo-tekton-testing.yaml:7]: there is a mismatch with the namespaces now. This says `insights-rules-tenant` but is inside the `insights-core-tenant` folder

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

File: auto-generated/cluster/stone-prd-rh01/tenants/insights-core-tenant/appstudio.redhat.com_v1beta2_integrationtestscenario_insights-puptoo-tekton-testing.yaml (hunk 3 above)
Comment (chenlizhong) [reply auto-generated/cluster/stone-prd-rh01/tenants/insights-core-tenant/appstudio.redhat.com_v1beta2_integrationtestscenario_insights-puptoo-tekton-testing.yaml]: Updated.

**Comment by chenlizhong** [comment]
@jsztuka @ernesgonzalez33   Updated the name of namespace. Please review again

**Review by ernesgonzalez33 — APPROVED** [review]

**Pull request merged**